apps/web-giddh/src/app/invoice/eWayBill/download-or-preview-eway/download-or-preview-eway.component.html

```diff
@@ -2,7 +2,7 @@
     <!-- <div *ngIf="isErrOccured$ | async">
         <p class="lead alC">Oops! Something went wrong.</p>
     </div> -->
-    <!-- <div class="loader mrT4" *ngIf="!showPdfWrap">
+    <!-- <div class="loader mr-t4" *ngIf="!showPdfWrap">
```

Comment:
remove commented code if not required, why we are updating commented code

Reply:
@rohit-walkover  this PR also contains code-designing part, but that was not completed, all files who has class = mrT4 is updated in this PR. so this commented code also got updated.